ludwig/features/numerical_feature.py

```diff
@@ -54,7 +59,7 @@ def inverse_transform(self, x: np.ndarray) -> np.ndarray:
         return x * self.sigma + self.mu
 
     @staticmethod
-    def fit_transform_params(column: np.ndarray, backend: "Backend") -> dict:
+    def fit_transform_params(column: np.ndarray, backend) -> dict:
```

Comment:
Can you keep the type hint but add a `noqa` instead? I believe it should be `noqa F821` but not 100% sure.

Reply:
Here and in the other type hint removals in this file.

Reply:
Done. 

Adding noqa bypasses the lint error, though I wonder if type hints causing linter errors is an indication of a suboptimal design.